jetty-websocket/websocket-jetty-api/src/main/java/org/eclipse/jetty/websocket/api/util/WSURI.java

```diff
@@ -104,22 +104,13 @@ public static URI toWebsocket(final URI inputUri) throws URISyntaxException
         Objects.requireNonNull(inputUri, "Input URI must not be null");
         String httpScheme = inputUri.getScheme();
         if ("ws".equalsIgnoreCase(httpScheme) || "wss".equalsIgnoreCase(httpScheme))
```

Comment:
Replace this line too using `HttpScheme.WS[S].is(...)`

Reply:
We can't really do this in the Jetty WebSocket API, we don't want to add a dependency on `jetty-http`.